### PR TITLE
use rawgit of PR branch for selenium tests -- manual input

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

#### Description

This is to discuss how we could input username and branch variables for a rawgit service of the bundle url for selenium tests.

### CODE REVIEW

File: test/setup.js.travis [hunk 1]
```diff
@@ -12,7 +12,10 @@ global.chromeBinaryPath = '/usr/bin/google-chrome-stable';
 global.screenshotDir = 'travis-screenshots';
 
 // URL for local bundle.js
-global.bundleUrl = 'https://localhost.daplie.me:8080/build/bundle.js';
+global.userName = 'freeCodeCamp';
+global.branch = 'master';
+global.bundleUrl = `https://rawgit.com/
+  ${global.userName}/testable-projects-fcc/${global.branch}/build/bundle.js`;
```
Comment (tbushman) [inline test/setup.js.travis]: ... so if I were submitting a new PR, for example, `fix/thing-to-fix`, I would push fixes to my fork before running tests, then change these variables to `global.userName = 'tbushman', global.branch = 'fix/thing-to-fix'`.

**Comment by tchaffee** [comment]
@tbushman I don't see any problem with these changes. It is a bit cumbersome for testing though because you have to push to your own repo after every change. Which is why I prefer serving the bundle locally. But I'm assuming this is useful to you? If so, I'm happy to merge it, but could you please also update the documentation before I merge it?

**Comment by tbushman** [comment]
@tchaffee This was mainly for discussion, but using this method I've determined that the drum machine PR #308 passes tests.

**Comment by tchaffee** [comment]
> This was mainly for discussion

I'm a little confused. Do you want to just delete this PR? It does seem useful as an alternative to local testing, no? It's ok with me whatever you want to do with this though. No pressure either way.

**Comment by tbushman** [comment]
@tchaffee Mainly look at PR #308 . You can use this method to show that it passes tests if you wish.

**Comment by tbushman** [comment]
Closing in favor of #311

**Pull request closed without merging**